src/main/java/customerrelationsmanagement/Emailer.java

```diff
@@ -248,4 +283,39 @@ public void sendMail(String recipientAddress, String messageSubject,
 		Transport.send(message);
 		System.out.println();
 	} // End processEmails
+	/**
+	 * Returning a statement for recommendations that will be sent
+	 * in a email after an order has been confirmed
+	 *
+	 * @param limit
+	 *  max of suggestive products
+	 * @param orderId
+	 *  to find matching recommendations against the current order
+	 *
+	 * @return a string for the message body
+	 *
+	 * @throws SQLException if a parameter of the query is wrong or
+	 * the connection is interrupted
+	 */
+	@NotNull private String recommendProducts(int limit, String orderId)
```

Comment:
I didn't know the addition of this feature would look so clean, great job using mySQL to accomplish this